SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/GameController.cs

```diff
@@ -20,33 +31,105 @@ public class GameController : Controller
         /// <returns>list of games view</returns>
         public ActionResult Index()
         {
-            //if (Roles.IsUserInRole(@"employee"))
-            //{
-            //    return View(db.Games.ToList());
-            return RedirectToAction("GameManagement");
-            //}
-            //else if (Roles.IsUserInRole(@"member"))
-            //{
-            //    //return SearchGames view
-            //      return RedirectToAction("SearchGames");
-            //}
-            //else
-            //{
-            //    //return SearchGames view
-            //      return RedirectToAction("SearchGames");
-            //}
+            //User is employee, redirect to GameManagement
+            if (Roles.IsUserInRole(@"employee"))
```

Comment:
After I merged to master locally, I got an error here. Does this work pre-merge?